CMake/External_FFmpeg.cmake

```diff
@@ -47,6 +52,46 @@ else ()
   set(_FFmpeg_yasm --yasmexe=${fletch_YASM})
   list(APPEND ffmpeg_DEPENDS yasm)
 
+  # Should we try to point ffmpeg at zlib if we are building it?
```

Comment:
I think we will have no choice but to hope for the best here. In looking at configure --help, it seems FFmpeg is one of those wonderful packages who don't let you pass a path to packages when you --enable-pkg, it insists on autodetect.

I think we just need to deal with it if it becomes an issue. Most *nix based systems should have what they need. The only thing we might be able to do it, find_package(ZLib .. ) and enable/disable based on that. I think what we have now is ok until it isn't.